index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio Aron</title>
+</head>
+<body>
+    <header class="top-navigation">
+        <a href="#"><img src="./Image/John_Doe.png" class="log" alt=" My Logo"></a>
```

Comment:
**[OPTIONAL]** As this is your portfolio and the logo should have to be your name and please remove the image and replace it with the text of your name.